container_opts.go

```diff
@@ -14,10 +15,11 @@ type NewContainerOpts func(ctx context.Context, client *Client, c *containers.Co
 
 // WithRuntime allows a user to specify the runtime name and additional options that should
 // be used to create tasks for the container
-func WithRuntime(name string) NewContainerOpts {
+func WithRuntime(name string, options *types.Any) NewContainerOpts {
```

Comment:
Maybe take an interface here and we will do the `Any` marshal with the types package

Reply:
updated